Formula/icpld.rb

```diff
@@ -0,0 +1,38 @@
+class Icpld < Formula
+  desc "Connection monitor to track your network connection performance."
+  homepage "http://www.ibiblio.org/icpld/"
+  url "http://www.ibiblio.org/icpld/download/icpld-1.1.5.tar.gz"
+  sha256 "9e54c33ce3b2ff2b72f5772aca02ef9f34807b5456054c328eed5b5d61b3a7c1"
+
+  option "with-ipv6", "If you plan on using icpld with IPv6 network."
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. We'll probably also need to have this patch accepted before we can merge this. Thanks!